### PR TITLE
Added regression tests for sql delete

### PR DESCRIPTION
**Pull request by gordonguthrie**

### CODE REVIEW

File: priv/riak_shell/riak_shell_regression1.log [hunk 1]
```diff
@@ -44,7 +44,7 @@ GeoCheckin
 {{command, "insert into GeoCheckin (myfamily, myseries, time, weather, temperature) values ('family1','series1',10,'hail',38.1);\n"}, {result, ""}}.
 {{command, "select time, weather, temperature from GeoCheckin where myfamily='family1' and myseries='seriesX' and time > 10 and time < 1000;\n"}, {result, ""}}.
 {{command, "select * from GeoCheckin;\n"}, {result, "Error (1001): The query must have a where clause."}}.
-{{command, "select * from GeoCheckin where myfamily = 'family1' and myseries = 'series1' and time >= 1420113600000 and time <= 1420119300000;\n"}, {result, []}}.
+{{command, "select * from GeoCheckin where myfamily = 'family1' and myseries = 'series1' and time >= 1420113600000 and time <= 1420119300000;\n"}, {result, "Error (1001): Too many subqueries (7)"}}.
```
Comment (hmmr) [inline priv/riak_shell/riak_shell_regression1.log]: Does this error still encountered in the wild?

File: priv/riak_shell/riak_shell_regression1.log [hunk 2]
```diff
@@ -80,133 +80,14 @@ family1,series1,1970-01-01T00:00:00.007Z,cloudy,27.9
...
-    clear_history, h, history, show_history
-
-You can get more help by calling help with the
-extension name and function name like 'help history clear_history;'
-
-For SQL help type 'help SQL'"}}.
-{{command, "help history h; "}, {result, "You can rerun a command by finding the command in the history list
-with `show_history;` and using the number next to it as the argument
-to `history` or `h`: `history 3;` or `h 3;` for example."}}.
+{{command, "delete from deletetable where myfamily = 'berl' and myseries = 'saf' and time = 3;"}, {result, ""}}.
```
Comment (hmmr) [inline priv/riak_shell/riak_shell_regression1.log]: A DELETE query with incomplete values in WHERE clause would be appropriate.

**Review by hmmr — CHANGES_REQUESTED** [review]
A failing DELETE query would be good.

**Review by hmmr — APPROVED** [review]

**Comment by hmmr** [comment]
+1

File: priv/riak_shell/riak_shell_regression1.log [hunk 3]
```diff
@@ -78,8 +79,14 @@ family1,series1,1970-01-01T00:00:00.007Z,cloudy,27.9
 {{command, "INSERT INTO GeoCheckin2 (region, state, time, weather, temperature) VALUES ('South Atlantic', 'South Carolina', 1452252523189, 'snowing', 19.2);\n"}, {result, "Inserted 1 row."}}.
 {{command, "SELECT 555, 1.1, 1e1, 1.123e-2 from GeoCheckin2 WHERE time > 1452252523182 AND time < 1452252543182 AND region = 'South Atlantic' AND state = 'South Carolina';\n"}, {result, "555,1.1,10.0,0.01123
 555,1.1,10.0,0.01123
-555,1.1,10.0,0.01123
```
Comment (paegun) [inline priv/riak_shell/riak_shell_regression1.log:81]: This line should not have been removed. There should be 3 rows returned (1st one is the columns). Perhaps it's a bit confusing b/c of the intentionally hardcoded values in the SELECT clause, but this is testing A) the number of rows returned; B) that values of each of the types in the SELECT clause are properly returned.

**Comment by paegun** [comment]
+1 on the feature under test.

The issue requiring re-adding the line from regression log is due to the columns of a SELECT statement containing only literal values being reported as what looks like a row. When we have aliasing via `AS` this regression log line can be made more clear, making it less likely to confuse the next person.

**Comment by hmmr** [comment]
+1

**Pull request merged**